<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_Configuration</class>
 <widget class="QDialog" name="dialog_Configuration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>458</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>549</width>
    <height>458</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TETECO - Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="teteco.qrc">
    <normaloff>:/teteco.png</normaloff>:/teteco.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>531</width>
       <height>88</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>531</width>
       <height>88</height>
      </size>
     </property>
     <property name="title">
      <string>Audio devices configuration</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>56</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Device IN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Device OUT</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Devices_IN">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>401</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Devices_OUT">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>401</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>522</width>
       <height>87</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>522</width>
       <height>87</height>
      </size>
     </property>
     <property name="title">
      <string>Speex Configuration</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Mode</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Quality</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_SpeexMode">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_SpeexQuality">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>54</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
     <zorder>comboBox_SpeexMode</zorder>
     <zorder>spinBox_SpeexQuality</zorder>
     <zorder>label_7</zorder>
     <zorder>label_6</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>531</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>525</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Server Configuration</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>357</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Listening Port</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_ServerPort">
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>22022</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Maximum Transfer Rate for Files</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_Transfer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>KB/s</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="minimumSize">
      <size>
       <width>522</width>
       <height>61</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>522</width>
       <height>61</height>
      </size>
     </property>
     <property name="title">
      <string>File reception</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>501</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Directory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Directory"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Directory">
         <property name="text">
          <string>Choose</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>188</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Accept">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="teteco.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Accept</sender>
   <signal>released()</signal>
   <receiver>dialog_Configuration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
